### PR TITLE
Improve the item-value length limit configuration document

### PR DESCRIPTION
**Pull request by klboke**
Improve the item-value length limit configuration document

### CODE REVIEW

File: docs/zh/deployment/distributed-deployment-guide.md [hunk 1]
```diff
@@ -1226,6 +1226,13 @@ http://5.5.5.5:8080/eureka/,http://6.6.6.6:8080/eureka/
 
 默认配置是20000。
 
+#### 3.2.5.1 namespace.value.length.limit.override - namespace 的配置项 value 最大长度限制
+
+此配置用来覆盖 `item.value.length.limit` 的配置，做到细粒度控制 namespace 的 value 最大长度限制，配置的值是一个 json 格式，json 的 key 为 namespace 在数据库中的 id 值，格式如下：
+```
+namespace.value.length.limit.override = {1:200,3:20}
+```
+以上配置指定了 ApolloConfigDB.Namespace 表中 id=1 的 namespace 的 value 最大长度限制为 200，id=3 的 namespace 的 value 最大长度限制为 20
```
Comment (nobodyiam) [inline docs/zh/deployment/distributed-deployment-guide.md:1235]: it's better to add a new line after the content
```suggestion
以上配置指定了 ApolloConfigDB.Namespace 表中 id=1 的 namespace 的 value 最大长度限制为 200，id=3 的 namespace 的 value 最大长度限制为 20

```

Comment (nobodyiam) [reply docs/zh/deployment/distributed-deployment-guide.md:1235]: btw, we also need to update the CHANGES.md

Comment (klboke) [reply docs/zh/deployment/distributed-deployment-guide.md:1235]: To improve the documentation, do I need to update CHANGES.md?

**Review by nobodyiam — APPROVED** [review]
LGTM

**Pull request merged**